### ISSUE TITLE
Could not get annual data

### ISSUE THREAD

**Issue by aarnaud** [titilambert/pyhydroquebec#3]
````
# ~/.local/bin/pyhydroquebec -u xxxxxx -c xxxxx -p xxxxx    
Could not get annual data
````

**Comment by titilambert** [comment]
Hello, Thanks for reporting.
* Which version are you using ?
* Could you retry today ? something hydroquebec website has some issues

**Comment by aarnaud** [comment]
I retry with 2.0.0

```
Task exception was never retrieved
future: <Task finished coro=<HydroQuebecClient.fetch_data() done, defined at /home/aarnaud/.local/lib/python3.5/site-packages/pyhydroquebec/client.py:276> exception=PyHydroQuebecError('Could not get annual data',)>
Traceback (most recent call last):
  File "/usr/lib/python3.5/asyncio/tasks.py", line 239, in _step
    result = coro.send(None)
  File "/home/aarnaud/.local/lib/python3.5/site-packages/pyhydroquebec/client.py", line 301, in fetch_data
    annual_data = yield from self._get_annual_data(p_p_id)
  File "/home/aarnaud/.local/lib/python3.5/site-packages/pyhydroquebec/client.py", line 221, in _get_annual_data
    raise PyHydroQuebecError("Could not get annual data")
pyhydroquebec.client.PyHydroQuebecError: Could not get annual data
```

I think, i don't have annual data on Hydro Quebec because, I'm customer since few month only.

**Comment by titilambert** [comment]
Ho !
thanks ! I will not raise an exceptio if you don't have annual data !

**Comment by titilambert** [comment]
@aarnaud  I just push a new commit, could do retry with the last upstream version ?
Thanks !

**Comment by aarnaud** [comment]
@titilambert it works since https://github.com/titilambert/pyhydroquebec/commit/d66f6035b92baed4869779fd92de95191d06bc54

**Comment by titilambert** [comment]
@aarnaud Thanks !
Released in 2.0.1, I will be update the home assistant plugin
